models/repo_avatar.go

```diff
@@ -109,7 +109,7 @@ func (repo *Repository) AvatarLink() string {
 func (repo *Repository) avatarLink(e db.Engine) string {
 	link := repo.relAvatarLink(e)
 	// link may be empty!
-	if len(link) > 0 {
+	if len(link) > 1 {
 		if link[0] == '/' && link[1] != '/' {
```

Comment:
I think you might need to handle the case where `link := "/"`

Reply:
If the avatar's link is `/`, that would be equal to that the repo doesn't have a avatar.

Reply:
yup but as it stands you would end up returning `/` which would be incorrect when Gitea is on a suburl

Reply:
Yeah that's correct, the only time that it can happen is if the user explicitly set the `FallbackImage` URL to `/`. What do you suggest we do return instead?

Reply:
setting.AppURL + "/" is probably the correct thing